heat/sparse/manipulations.py

```diff
@@ -2,16 +2,53 @@
 from __future__ import annotations
 
 from heat.sparse.dcsr_matrix import DCSR_matrix
-
+from heat.sparse.factories import sparse_csr_matrix
 from ..core.memory import sanitize_memory_layout
 from ..core.dndarray import DNDarray
 from ..core.factories import empty
 
 __all__ = [
     "todense",
+    "to_sparse",
 ]
```

Comment:
@samadpls just to clarify, I think we should name these functions consistently (see below). The scipy `sparse.crs_matrix` has a method `todense()` so we should offer that as an alias. 

Do numpy ndarrays  have a `tosparse()` method? I don't think so but I might be wrong.  

```suggestion
__all__ = [
    "to_dense",
    "to_sparse",
]
```

Reply:
Thanks for the clarification! I've updated the code to keep the naming consistent. 😋